react-components/src/components/RevealToolbar/RuleBasedOutputsButton.tsx

```diff
@@ -22,35 +22,57 @@ import { useTranslation } from '../i18n/I18n';
 import { useFetchRuleInstances } from '../RuleBasedOutputs/hooks/useFetchRuleInstances';
 import { type AssetStylingGroup } from '../..';
 
+type EmptyRule = {
+  rule: any;
```

Comment:
Could we use a more precise type than `any` here?

Reply:
ops.. forgot to push this

https://github.com/cognitedata/reveal/pull/4453/commits/a1d04afb98d76e7e109372cd20be758f5ec689c6